docs/REST API/rest-api.md

```diff
@@ -4969,20 +4968,18 @@ Updates the specified label. Note: Label queries and platforms are immutable. To
 | id          | integer | path | **Required**. The label's id. |
 | name        | string  | body | The label's name.             |
 | description | string  | body | The label's description.      |
-| hosts       | array   | body | If updating a manual label: the list of host identifiers (`hardware_serial`, `uuid`, `osquery_host_id`, `hostname`, or `name`) the label will apply to. |
+| hosts       | array   | body | If updating a manual label: the list of host identifiers (`hardware_serial`, `uuid`, or `hostname`) the label will apply to. **Note: The provided list fully replaces the previous list.** |
```

Comment:
Added this note to clarify existing behavior that the new list fully replaces the previous list based on this Slack [thread](https://fleetdm.slack.com/archives/C019WG4GH0A/p1730847020034899).